Gemfile

```diff
@@ -42,6 +42,7 @@ group :development, :test do
   gem 'launchy'
   gem 'shoulda-matchers', '~> 3.0'
   gem 'fabrication'
+  gem 'faker'
```

Comment:
I pulled it down and played around with it. I kept getting an uninitialized constant Faker::HarryPotter error. I found a solution. It has to look this in the gem file:  gem 'faker', github: 'stympy/faker.git'